core/api/src/graphql/public/root/query/payout-speeds.ts

```diff
@@ -0,0 +1,18 @@
+import { OnChain } from "@/app"
+
+import { GT } from "@/graphql/index"
+import { mapError } from "@/graphql/error-map"
+import PayoutSpeedsObject from "@/graphql/public/types/object/payout-speeds"
+
+const PayoutSpeedsQuery = GT.Field({
+  type: GT.NonNullList(PayoutSpeedsObject),
+  description: "Returns the list of available payout queues for on-chain payouts",
```

Comment:
queues is used internally, this should be something like "Returns the list of available speeds for on-chain payments"